apps/namada-interface/src/components/Toast/toast.components.ts

```diff
@@ -0,0 +1,60 @@
+import styled from "styled-components/macro";
+import { motion } from "framer-motion";
+
+export const Wrapper = styled(motion.div)<{ index: number }>`
+  display: flex;
+  justify-content: space-between;
+  position: fixed;
+  transition: top 0.5s;
+  top: ${(props) => 100 + props.index * 70}px;
+  right: 15px;
+  width: 360px;
+  z-index: 10;
```

Comment:
Was not sure about value :)

Reply:
Yeah, we have not defined them in the design system. With `z-index: 10;`, the select element goes on top of this so, maybe to play it safe as long as we define them we could set it to:
```suggestion
  z-index: 9999;
```